src/components/TodoFooter.tsx

```diff
@@ -0,0 +1,48 @@
+import classNames from 'classnames';
+import React, { useContext } from 'react';
+import { Status } from '../types/Status';
+import { TodoContext } from '../contexts/TodoContext';
+
+type Props = {
+  completed: number;
+};
+
+export const TodoFooter: React.FC<Props> = ({ completed }) => {
+  const { status, changeStatus } = useContext(TodoContext);
+
+  const handleChangeStatus = (statusValue: Status) => {
+    changeStatus(statusValue);
+  };
+
+  return (
+    <footer className="todoapp__footer" data-cy="Footer">
+      <span className="todo-count" data-cy="TodosCounter">
+        {`${completed} items left`}
+      </span>
+
+      <nav className="filter" data-cy="Filter">
+        {Object.values(Status).map(statusValue => (
+          <a
```

Comment:
Try to re-write it like this. The best practice is to use map with `ul` and `li` components
```suggestion
      <nav className="filter" data-cy="Filter">
      	<ul>
	        {Object.values(Status).map(statusValue => (
	        	<li>
	          	<a	
```